openmdao/approximation_schemes/approximation_scheme.py

```diff
@@ -20,18 +20,18 @@ class ApproximationScheme(object):
 
     Attributes
     ----------
-    _approx_groups: list
+    _approx_groups : list
```

Comment:
Changes to these files are missing:

Missing changes you made to:
  openmdao/code_review/test_lint_attributes.py
  openmdao/test_suite/groups/sin_fitter.py
  openmdao/test_suite/test_examples/beam_optimization/multipoint_beam_group.py 
  openmdao/test_suite/test_examples/beam_optimization/multipoint_beam_stress.py 
  openmdao/test_suite/test_examples/cannonball/test_euler_integration.py

Reply:
Fixed